### PR TITLE
Task 11

### PR DESCRIPTION
**Pull request by danchikov-a**
Added possibility to sort, filter and paginate

### CODE REVIEW

File: resources/views/products.blade.php [hunk 1]
```diff
@@ -71,14 +100,16 @@
                 </td>
                 <td>
                     <form action="{{route('productsDestroy', $product->id)}}" method="post">
-                        <input class="btn btn btn-outline-danger" type="submit" value="Delete" />
+                        <input class="btn btn btn-outline-danger" type="submit" value="Delete"/>
```
Comment (vladislavzubetcs) [inline resources/views/products.blade.php]: `<button type="submit" class="btn btn btn-outline-danger">Delete</button>`

File: resources/views/products.blade.php [hunk 2]
```diff
@@ -71,14 +100,16 @@
                 </td>
                 <td>
                     <form action="{{route('productsDestroy', $product->id)}}" method="post">
```
Comment (vladislavzubetcs) [inline resources/views/products.blade.php]: {{ route('productsDestroy', $product->id) }}

File: resources/views/products.blade.php [hunk 3]
```diff
@@ -71,14 +100,16 @@
...
                         @method('delete')
                         @csrf
                     </form>
                 </td>
             </tr>
         @endforeach
-
         </tbody>
     </table>
+    <div class="d-flex">
```
Comment (vladislavzubetcs) [inline resources/views/products.blade.php]: Move this pagination to partials folder. And after that `@include('partials.pagination', $products->links()')`

File: resources/views/products.blade.php [hunk 4]
```diff
@@ -39,7 +39,36 @@
         @if($errors->has('cost'))
```
Comment (vladislavzubetcs) [inline resources/views/products.blade.php]: `@error('cost')`

**Review by vladislavzubetcs — CHANGES_REQUESTED** [review]

File: resources/views/products.blade.php (hunk 1 above)
Comment (danchikov-a) [reply resources/views/products.blade.php]: Fixed

File: resources/views/products.blade.php (hunk 2 above)
Comment (danchikov-a) [reply resources/views/products.blade.php]: Fixed

File: resources/views/products.blade.php (hunk 3 above)
Comment (danchikov-a) [reply resources/views/products.blade.php]: Fixed

File: resources/views/products.blade.php (hunk 4 above)
Comment (danchikov-a) [reply resources/views/products.blade.php]: Fixed

**Pull request merged**